docs/did/asset-did-spec.md

```diff
@@ -0,0 +1,152 @@
+This is required to render the resulting string a valid URI according to the [DID Core Specification][did-core-spec].
+
+Chain and asset namespaces are defined with Improvement Proposals and are stored under the [CAIP repository][caip-repo].
+For instance, the namespaces for [EIP155][caip-3-spec] (Ethereum-based) and [BIP122][caip-4-spec] (Bitcoin-based) chains as well as for [SLIP44][caip-20-spec] (native fungible tokens), [ERC20][caip-21-spec] (smart contract based tokens), and [ERC721][caip-22-spec] (NFTs) assets have already been defined.
+
+## Method Operations
+
+### Create an Asset DID
+
+An Asset DID as such is not directly created, but it is generated at any time from a specific asset on as specific network.
+Because of the chain agnostic nature of Asset DIDs, this process can take place on any blockchain that is identifiable by a [CAIP-2 Identifier][caip-2-spec].
+
+Furthermore, as far as Asset DIDs are concerned, the assets identified do not need to already exist.
+For instance, a "right to create" a new NFT collection on a given blockchain for a user could be encoded in a [Verifiable Credential][vc-spec] where the identifier of the asset is an Asset DID.
+
+<!-- TODO: Investigate the usage of the `alsoKnownAs` property when supporting asset transfers -->
+
+### Update an Asset DID
+
+The Asset DID method does not support direct updates, being a purely generative method, but its resolution reflects any changes to the underlying asset or to its credentials.
+
+### Deactivate an Asset DID
+
+The Asset DID method does not support deactivations, being a purely generative method.
+
+## Asset DID Resolution
+
+The information required to resolve an Asset DID is the following:
+
+- The **chain** on which the asset lives, e.g., the Ethereum mainnet
+- The **namespace** defining the asset, e.g., the ERC721 namespace
+- The **reference** to the asset class within the defined namespace, e.g., the smart contract address for an NFT collection
+- [OPTIONAL] The unique **identifier** of the asset instance within the asset class, e.g., the NFT ID within a given collection
+
+The resolution process then follows a hierarchical structure.
+
+The chain identifier is decomposed into its namespace and reference.
+Then, the resolver for the specified namespace is used to resolve the specified reference.
+
+The same process is followed for the asset identifier.
+The identifier is decomposed into its namespace, reference, and optional ID.
+Then, the resolver for the specified asset namespace is used to resolve the asset reference and ID.
+
+As an example, the returned DID Document for [one of the CryptoKitties NFTs](https://opensea.io/assets/ethereum/0x06012c8cf97bead5deae237070f9587f8e7a266d/634446), identified with the Asset DID `did:asset:eip155:0.erc721:0x06012c8cf97bead5deae237070f9587f8e7a266d:634446`, is the following:
```

Comment:
eip155:0 should be eip155:1 if you're referring to Eth Mainnet.  TBH I'm not sure what 0 represents, but it should probably represent either "all networks" or "network unknown"-- you've inspired me to open an issue haha

Reply:
Good catch! That was definitely a typo 😄 Fixed in https://github.com/KILTprotocol/specifications/pull/3/commits/23e4194eb066f46e74765914dffba9a84fc0853b